docs/en/advanced/neural_networks.md

```diff
@@ -4,29 +4,29 @@
 This is an experimental module. All flying at your own risk.
 :::
 
-There are several reasons you might want to use neural networks inside of PX4. This documentation together with an example module will help you to get started with this. The code is made to run directly on an embedded flight controller (FCU), but it can easily be modified to run on a more powerful companion computer as well.
+There are several reasons you might want to use neural networks inside of PX4, such as making controllers for experimental drone morphologies or computer vision taks and so on. This documentation together with an example module will help you to get started with this. The code is made to run directly on an embedded flight controller (FCU), but it can easily be modified to run on a more powerful companion computer as well.
```

Comment:
```suggestion
There are several reasons you might want to use neural networks inside of PX4, such as making controllers for experimental drone morphologies or computer vision tasks and so on. This documentation together with an example module will help you to get started with this. The code is made to run directly on an embedded flight controller (FCU), but it can easily be modified to run on a more powerful companion computer as well.
```